proton/api.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, api_url, appversion="Other"):
         self.s.headers['x-pm-appversion'] = appversion
 
     def api_request(self, endpoint, jsondata=None, additional_headers=None, method=None):
+        self.s.mount(self.__api_url, TLSPinning())
```

Comment:
This needs to be done for every URL?

Reply:
This happens for every `api_request`, so yes.

Reply:
I mean: I see that it happens for every `api_request` :sweat_smile: What I wonder is if it needs to be.
I think it can be be simply called at constructor time initialization?
```python
for domain in tls_pinned_domain_list:
    self.s.mount("https://" + domain, TLSPinning())
```

Reply:
If we are going to use one url per session then I guess yes, makes more sense.